src/vs/workbench/services/suggest/browser/media/suggest.css

```diff
@@ -117,8 +117,12 @@
 	touch-action: none;
 }
```

Comment:
@tyriar I tried to put this in a new `terminalSuggest.css` file, along with a `.terminal` selector, importing it as `import './media/terminalSuggest.css';` in the `terminal.contrib` file, but it wasn't getting applied.

Reply:
Importing should work, I think sometimes the watch might not trigger correctly, so you could try restart the build? We definitely don't want to add css to suggest/ that it doesn't know about.

Reply:
figured it out by putting it at the top of `terminalSymbolIcons.js`, which gets imported in `terminal.suggest.contrib`. putting a `/media/.css` file directly in there, regardless of which it is, fails.

<img width="825" alt="Screenshot 2025-06-17 at 7 33 50 PM" src="https://github.com/user-attachments/assets/e5b761c2-a509-49f1-9c2d-b5fd88c49de6" />